api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,215 @@
 			<column name="uuid" type="char(38)" />
 		</addColumn>
 	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1">
+		<preConditions onFail="MARK_RAN">
+			<foreignKeyConstraintExists foreignKeyName="member_patient"/>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="member_patient" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.1">
+		<preConditions onFail="MARK_RAN">
+			<foreignKeyConstraintExists foreignKeyName="parent_cohort"/>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="parent_cohort" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.2">
+		<preConditions onFail="MARK_RAN">
+			<primaryKeyExists tableName="cohort_member"/>
+		</preConditions>
+		<comment>Dropping primary key for cohort_member</comment>
+		<dropPrimaryKey tableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="cohort_member_id"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="cohort_member_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="start_date"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="start_date" type="datetime"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610242135-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">
+					SELECT COUNT(*) from cohort_member
+				</sqlCheck>
+			</not>
+		</preConditions>
+		<update tableName="cohort_member">
+			<column name="start_date" type="DATETIME" value="select date_created from cohort"/>
```

Comment:
I had never done this before, did you test to confirm that it works? I used to think valueComputed is what supports this